drivers/clock_control/clock_control_mcux_ccm_rev2.c

```diff
@@ -74,6 +74,13 @@ static int mcux_ccm_get_subsys_rate(const struct device *dev,
 #endif
 #endif
 
+#ifdef CONFIG_DT_HAS_NXP_MCUX_I3C_ENABLED
```

Comment:
Why use the DT Kconfig versus CONFIG_I3C_MCUX?

Reply:
Sorry, I didn't think too much about it. CONFIG I3C MCUX looks more appropriate.